PySDM/dynamics/collisions/coalescence_efficiencies/straub2010.py

```diff
@@ -47,4 +47,4 @@ def __call__(self, output, is_first_in_pair):
         self.arrays["We"].divide_if_not_zero(self.arrays["Sc"])
         self.arrays["We"] *= -1.15
 
-        output[:] = np.exp(self.arrays["We"])
+        output[:] = np.exp(self.arrays["We"])  # TODO #976
```

Comment:
Should we do the same "fill" call here?

Reply:
this line is bogus as it uses NumPy, which e.g. will not work on the GPU backend which does not use NumPy at all. We need to introduce some new backend method that implements `exp`

Reply:
and even on the CPU backend, this line causes unnecessary allocation (each time the code is called) of a temporary array for the result of `exp`

Reply:
since a TODO label was added, I'm marking it as resolved (i.e., to be resolved in another PR)